src/reducers/events.js

```diff
@@ -1,18 +1,82 @@
 import {combineReducers} from 'redux';
+const _ = require('lodash');
 const events_test_data = [{name: "Soccer game", location: "Empire Field, Vancouver", date: new Date("2021-07-02"), genre: ['sports', 'outdoor', 'soccer'], creator: {name: "John Doe"}, participants: [{name: "John Doe"}, {name: "Susan Thompson"}, {name: "Pablo Johnson"}, {name: "Alex Tim"}], description: "A soccer game for anyone to join", comments: [{user: {name: "Alex Tim"}, text: "Sounds like fun. I'll be there"}, {user: {name: "Harriet Norman"}, text: "Can't make it this time, I'll be there for the next one"}]}, 
                           {name: "Basketball game", location: "Kits Beach, Vancouver", date: new Date("2021-07-05"), genre: ['sports', 'indoor', 'basketball'], creator: {name: "Jane Doo"}, participants: [{name: "Susan Thompson"}, {name: "Gordon Lighthouse"}, {name: "Heather Baller"}], description: "1 v 1 pickup basketball at the Kits beach courts. Playing king's court and aimed at intermediate level players. $5 to enter and winning team keeps the money", comments: [{user:{name: "Heather Baller"}, text: "I'm init to winit"}]}, 
                           {name: "MTG Tournament", location: "Vancouver Convention Centre", date: new Date("2020-01-04"), genre: ['games', 'card games', 'tournaments', 'magic'], creator: {name: "Joan Smith"}, participants: [{name: "Daniel Boyce"}, {name: "Carol Wisnewski"}, {name: "Samuel Zubrus"}, {name: "Gary Lee"}, {name: "Andrew Gergich"}, {name: "Jennifer Quincey"}, {name: "Joel Quincey"}, {name: "Joel Harland"}, {name: "Carla Gregg"}], description: "Magic The Gathering tournament. Winner takes home $25,000 grand prize", comments: []}];
 
 
+
+
+const addParticipant = (user, event, events) => {
+        let eventIndex = events.findIndex((element) => {return _.isEqual(element,event)});
+        if (!JSON.stringify(event.participants).includes(JSON.stringify(user))) {
+            event.participants = event.participants.concat(user);
+        }
+        if (eventIndex+1) {
```

Comment:
Is this suppose to be plus 1?

Reply:
Yes I think so. The first Index would be 0 which would evaluate to false when we want it to be true no?